cypress/integration/ansiLogs/ansi_logView.ts

```diff
@@ -200,3 +200,64 @@ describe("expanding collapsed rows", () => {
     cy.dataCy("collapsed-row-1-4").should("exist");
   });
 });
+
+describe("Sharing lines", () => {
+  const logLink =
+    "/evergreen/spruce_ubuntu1604_test_2c9056df66d42fb1908d52eed096750a91f1f089_22_03_02_16_45_12/0/task";
+
+  beforeEach(() => {
+    cy.visit(logLink);
+  });
+
+  it("should present a share button with a menu when a line is selected", () => {
+    cy.dataCy("line-index-1").click();
+    cy.dataCy("sharing-menu-button").should("be.visible");
+    cy.dataCy("sharing-menu-button").click();
+    cy.dataCy("sharing-menu").should("be.visible");
+  });
+  it("shift+click selecting a range of lines should automatically open the sharing menu", () => {
+    cy.dataCy("line-index-1").click();
+    cy.dataCy("line-index-10").click({ shiftKey: true });
+    cy.dataCy("sharing-menu").should("be.visible");
+  });
+  it("should be able to copy the selected lines as JIRA format", () => {
+    cy.dataCy("line-index-1").click();
+    cy.dataCy("line-index-2").click({ shiftKey: true });
+    cy.dataCy("sharing-menu").should("be.visible");
+    cy.contains("Copy selected contents").should("be.visible");
+    cy.contains("Copy selected contents").click();
+    cy.validateToast("success", "Copied 2 lines to clipboard", true);
+    cy.window().then((win) => {
+      win.navigator.clipboard.readText().then((text) => {
+        expect(text).to.eq(
+          `{noformat}\n[2022/03/02 17:01:58.587] Task logger initialized (agent version 2022-02-14 from 00a4c8f3e8e4559cc23e04a019b6d1725c40c3e5).\n...\n[2022/03/02 17:02:01.610] e391612 EVG-16049 Update spruce project page for admin only variables (#1114)\n[2022/03/02 17:02:01.610] 04a52b2 EVG-15959 Fix rerender method in test utils (#1118)\n...\n[2022/03/02 17:05:21.050] running setup group because we have a new independent task\n{noformat}`
```

Comment:
`{noformat}` is part of the jira string formatting. https://github.com/evergreen-ci/parsley/blob/314a94f5881ee4695ae8cbed4ca9d25bb0a41c61/src/utils/string/index.ts#L23

Reply:
This is preexisting logic from the lobster days. Server engineers rely on this since copying log lines into Jira tickets is a common workflow. Without the formatting characters, the lines won't be rendered correctly in jira. 
<img width="345" alt="image" src="https://github.com/evergreen-ci/parsley/assets/4605522/13e20900-62fc-410b-9ae6-22758def76ef">

Reply:
We can open a ticket to investigate making this more generic in the future if it comes up.